awx/main/models/credential/injectors.py

```diff
@@ -24,27 +24,32 @@ def gce(cred, env, private_data_dir):
         'type': 'service_account',
         'private_key': cred.get_input('ssh_key_data', default=''),
         'client_email': username,
-        'project_id': project
+        'project_id': project,
+        # need token uri for inventory plugins
+        # should this really be hard coded? Good question.
+        'token_uri': 'https://accounts.google.com/o/oauth2/token',
     }
+
     handle, path = tempfile.mkstemp(dir=private_data_dir)
     f = os.fdopen(handle, 'w')
-    json.dump(json_cred, f)
+    json.dump(json_cred, f, indent=2)
```

Comment:
Just more readable?

Reply:
yes, these kinds of tests are new, so looking at the files wasn't really a thing before.